packages/proxy/src/providers/bedrock.ts

```diff
@@ -498,9 +498,11 @@ function translateTools(tools: ChatCompletionTool[]): ToolConfiguration {
 export async function fetchConverse({
   secret,
   body,
+  signal,
 }: {
   secret: APISecret;
   body: Record<string, unknown>;
+  signal?: AbortSignal;
```

Comment:
im guessing the aws sdk doesnt let you thread this through?

Reply:
it actually does. good catch